AsyncDisplayKit/ASCellNode.h

```diff
@@ -24,6 +24,16 @@ typedef NSUInteger ASCellNodeAnimation;
  * @param sizeChanged `YES` if the node's `calculatedSize` changed during the relayout, `NO` otherwise.
  */
 - (void)nodeDidRelayout:(ASCellNode *)node sizeChanged:(BOOL)sizeChanged;
+
+@optional
+/**
+ * Notifies the delegate that the specified cell node has scrolled
+ *
+ * @param node: A node informing the delegate about the scroll
+ * @param cellFrame: The frame of the cell that has just scrolled
```

Comment:
add to the cellFrame - "...scrolled, in the coordinate system of the scroll view (this is different than the ASCellNode's frame).  Use this value for efficiency instead of convertRect: from the ASCellNode to the UIScrollView."